ambry-mysql/src/main/java/com/github/ambry/mysql/MySqlUtils.java

```diff
@@ -114,13 +115,15 @@ public static class DbEndpoint {
     private final boolean isWriteable;
     private final String username;
     private final String password;
+    private final MySqlNamedBlobDbConfig.SSLMode sslMode;
 
-    public DbEndpoint(String url, String datacenter, boolean isWriteable, String username, String password) {
+    public DbEndpoint(String url, String datacenter, boolean isWriteable, String username, String password, MySqlNamedBlobDbConfig.SSLMode sslMode) {
```

Comment:
Let's also add a new DbEndpoint constructor to not include SSLMode as the last parameter. If it's missing, we just NONE.

In this way, you don't have to change all the files that create DbEndpoint with the old method.